### ISSUE TITLE
Results of loss on SimCLR notebook

### ISSUE THREAD

**Issue by aynesss** [sayakpaul/SimCLR-in-TensorFlow-2#6]
Please, can you give us your results of Loss at each epoch in SimCLR notebook 
epoch: 1 loss: 0.751
  5%|████████                                                                                                                                                          | 10/200 [1:36:22<30:37:01, 580.11s/it]epoch: 11 loss: 0.706
 10%|████████████████▏                                                                                                                                                 | 20/200 [3:12:59<29:06:32, 582.18s/it]epoch: 21 loss: 0.707
 15%|████████████████████████▎                                                                                                                                         | 30/200 [4:49:41<27:24:37, 580.45s/it]epoch: 31 loss: 0.703
 20%|████████████████████████████████▍                                                                                                                                 | 40/200 [6:26:44<25:54:00, 582.75s/it]epoch: 41 loss: 0.701
 25%|████████████████████████████████████████▌                                                                                                                         | 50/200 [8:03:54<24:17:17, 582.92s/it]epoch: 51 loss: 0.701
 30%|████████████████████████████████████████████████▌                                                                                                                 | 60/200 [9:41:09<22:42:11, 583.80s/it]epoch: 61 loss: 0.700
 35%|████████████████████████████████████████████████████████▎                                                                                                        | 70/200 [11:18:29<21:05:40, 584.16s/it]epoch: 71 loss: 0.699
 40%|████████████████████████████████████████████████████████████████▍                                                                                                | 80/200 [12:55:51<19:26:37, 583.31s/it]epoch: 81 loss: 0.700
 45%|████████████████████████████████████████████████████████████████████████▍                                                                                        | 90/200 [14:32:46<17:46:44, 581.85s/it]epoch: 91 loss: 0.700

For me, here is what I found, Is that good?

**Comment by sayakpaul** [comment]
Please refer to the original paper first in order to get an idea of how the loss should behave as the training progresses. Also, this repository is accompanied by a blog post (mentioned at the very beginning in the README) which discusses the loss curves as well.

**Comment by aynesss** [comment]
Thanks a lot for your time, The problem is that my curve does not decrease until reach zero as shown in figure ( README ). So, in that case what I must do ? change those parameters : "tau = 0.1, h1 = 256, h2 = 128, h3 = 50" and do training again ? 
Your answer and help means a lot to me !!

**Comment by sayakpaul** [comment]
You can follow the suggestions from here listed in the README:

![image](https://user-images.githubusercontent.com/22957388/120891488-822d5d00-c626-11eb-807f-d94e1ef982ec.png)

You can also take a look at this implementation of SimCLR which is better than mine: https://keras.io/examples/vision/semisupervised_simclr.
